lib/matplotlib/collections.py

```diff
@@ -110,14 +111,10 @@ def __init__(self,
             where *onoffseq* is an even length tuple of on and off ink lengths
             in points. For examples, see
             :doc:`/gallery/lines_bars_and_markers/linestyles`.
-        capstyle : str, default: :rc:`patch.capstyle`
+        capstyle : `.CapStyle`-like, default: :rc:`patch.capstyle`
```

Comment:
@timhoffm This would be an example where including the individual possibilities would push us over a  single line.